packages/vue-client/src/components/boards/DefaultBoard.vue

```diff
@@ -0,0 +1,46 @@
+<!--
+This board is meant to be easy for a variety of variants to use.
+It supports features such as multicolor stones, non-grid boards,
+board color etc.
+-->
+
+<script setup lang="ts">
+import MulticolorGridBoard from "./MulticolorGridBoard.vue";
+import { Coordinate, indexToMove } from "@ogfcommunity/variants-shared";
+import MulticolorGraphBoard from "./MulticolorGraphBoard.vue";
+import {
+  DefaultBoardConfig,
+  DefaultBoardState,
+  MulticolorStone,
+} from "@ogfcommunity/variants-shared/src/lib/board_types";
```

Comment:
I've been trying to consistently import with "@ogfcommunity/variants-shared/" and then no path.

Reply:
Good catch! There were quite a few of these dang auto imports. I will fix before landing.

Reply:
related: https://github.com/govariantsteam/govariants/pull/323